repos/spack_repo/builtin/packages/py_dask/package.py

```diff
@@ -54,6 +55,7 @@ class PyDask(PythonPackage):
     depends_on("py-pyyaml@5.3.1:", type=("build", "run"), when="@2022.10.2:")
     depends_on("py-cloudpickle@1.1.1:", type=("build", "run"), when="@2021.3.1:")
     depends_on("py-cloudpickle@1.5.0:", type=("build", "run"), when="@2023.4.1:")
+    depends_on("py-cloudpickle@3.0.0:", type=("build", "run"), when="@2025.7.0:")
     depends_on("py-fsspec@0.6.0:", type=("build", "run"), when="@2021.3.1:")
     depends_on("py-fsspec@2021.09.0:", type=("build", "run"), when="@2023.4.1:")
     depends_on("py-toolz@0.8.2:", type=("build", "run"), when="@2021.3.1:")
```

Comment:
importlib-metadata dep is now conditional on Python version

Reply:
numpy and pandas min versions are updated

Reply:
pandas is `@2:` for `@2024.7.1: +dataframe` already so that should be covered off. Agreed I missed the numpy ty

Reply:
>importlib-metadata dep is now conditional on Python version

Do you think it is correct to assume `depends_on("py-importlib-metadata@4.13.0:", type=("build", "run"), when="@2023.4.0: ^python@:3.11")` is the correct approach?

Reply:
It's correct for 2025, you'll need to go back in time and see if it's correct for 2023 and 2024